pootle/apps/pootle_store/models.py

```diff
@@ -658,7 +659,7 @@ def sync(self, unit):
 
     def update(self, unit, user=None):
         """Update in-DB translation from the given :param:`unit`.
-
+        :param user: User to attribute updates to.
```

Comment:
This is an unrelated cleanup, so please create a separate commit for it and directly merge it to master.